### ISSUE TITLE
데모 영상 제출에 관하여

### ISSUE THREAD

**Issue by fjqlddb06** [swsnu/swppfall2020#241]
영상 용량이 커서 네이버 메일로 보낼려고 하는데 괜찮을까요?
그리고 따로 메일 제목이나 내용 지켜야할 양식이 있을까요?

**Comment by hy00nc** [comment]
네 어떤 방식으로 보내셔도 상관 없습니다. 메일 제목에는 [SWPP] TeamX Demo Video 정도로 팀 정보만 넣어주시면 감사하겠습니다. 
내용 양식 따로 없고 동영상만 보내시면 됩니다.
